_targets.R

```diff
@@ -10,18 +12,26 @@ source("1_fetch/src/get_site_data.R")
 source("3_visualize/src/map_sites.R")
 
 # Configuration
-states <- c('WI','MN','MI')
+states <- c('WI','MN','MI', 'IL')
 parameter <- c('00060')
 
 # Targets
 list(
   # Identify oldest sites
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
 
-  # TODO: PULL SITE DATA HERE
-  tar_target(wi_data, get_site_data(oldest_active_sites, states[1], parameter)),
-  tar_target(mn_data, get_site_data(oldest_active_sites, states[2], parameter)),
-  tar_target(mi_data, get_site_data(oldest_active_sites, states[3], parameter)),
+  # # TODO: PULL SITE DATA HERE
+  # tar_target(wi_data, get_site_data(oldest_active_sites, states[1], parameter)),
+  # tar_target(mn_data, get_site_data(oldest_active_sites, states[2], parameter)),
+  # tar_target(mi_data, get_site_data(oldest_active_sites, states[3], parameter)),
+
+  # Static branching
+  tar_map(
+    values = tibble(state_abb = states),
+    tar_target(nwis_data, get_site_data(oldest_active_sites, state_abb, parameter)),
+    tar_target(count, tally(nwis_data)),
+    tar_target(fig, plot(count))
```

Comment:
These two lines will be added in a future PR. There will be special functions that are added to your repo, so you can leave them commented out for now.